read_roi/_read_roi.py

```diff
@@ -316,9 +316,13 @@ def extract_basic_roi_data(data):
         overlayFontSize = get_uint16(data, hdr2Offset + HEADER_OFFSET['OVERLAY_FONT_SIZE'])
         imageOpacity = get_byte(data, hdr2Offset + HEADER_OFFSET['IMAGE_OPACITY'])
         imageSize = get_uint32(data, hdr2Offset + HEADER_OFFSET['IMAGE_SIZE'])
+        # abb, read group from file
+        group = get_byte(data, hdr2Offset + HEADER_OFFSET['AVAILABLE_BYTE1'])
```

Comment:
Please replace `HEADER_OFFSET['AVAILABLE_BYTE1']` with `HEADER_OFFSET['GROUP']` (both here, and of course also where the offset is defined. This is in keeping with our [source code reference](https://imagej.nih.gov/ij/developer/source/ij/io/RoiDecoder.java.html).